lib/src/internal/http/http_bucket.dart

```diff
@@ -9,86 +9,67 @@ import 'package:nyxx/src/internal/http/http_handler.dart';
 import 'package:nyxx/src/internal/http/http_request.dart';
 
 class HttpBucket {
-  // Rate limits
-  int _remaining = 10;
-  DateTime? _resetAt;
-  double? _resetAfter;
-  RestEventController get _events => _httpHandler.client.eventsRest as RestEventController;
+  static const String xRateLimitBucket = "x-ratelimit-bucket";
```

Comment:
Remove the x, I feel its not needed, switch it for the k prefix? This is the same for lines 12-16.

Reply:
x not removed here also lines below have 'k' in front (?) why? The point wass to make `xRateLimitBucket` into `rateLimitBucket`

Reply:
I was saying use the k prefix for constants but yes, could just have all prefix removed entirely